src/components/MoviesGrid.tsx

```diff
@@ -0,0 +1,51 @@
+import styled from 'styled-components';
+import React from 'react';
+import MovieCard from './MovieCard';
+import { Filter, MovieData, Genre } from '../model';
+
+export const movies = [{
+  value: 'DOCUMENTARY 1',
+  genre: Genre.Documentary,
+}, {
+  value: 'COMEDY 1',
+  genre: Genre.Comedy,
+}, {
+  value: 'DOCUMENTARY 2',
+  genre: Genre.Documentary,
+}, {
+  value: 'DOCUMENTARY 3',
+  genre: Genre.Documentary,
+}, {
+  value: 'COMEDY 4',
+  genre: Genre.Comedy,
+}, {
+  value: 'CRIME 2',
+  genre: Genre.Crime,
+}];
```

Comment:
As `movies` variable isn't supposed to be changed, you can mark it `as const`. It would help TS to statically analyse your code and provide better autocompletes etc.
```ts
export const movies = [
{
  value: 'DOCUMENTARY 1',
  genre: Genre.Documentary,
}, 
{
  value: 'COMEDY 1',
  genre: Genre.Comedy,
}
] as const;
```

Reply:
Fixed.